Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -1178,6 +1234,17 @@ namespace xr
 
             return true;
         }
+
+        void UpdatePoseData(xr::Pose& targetPose, XrPosef& sourcePose)
```

Comment:
Doesn't seem to be a reason for this to be a member function. Consider making this `static` or a free standing function.

Reply:
I made it a member function since it fits with the utility of the other member functions (like PopulateDepthInfoView)

Reply:
I think I'll keep it as-is to fit with the others.